oak_runtime/src/lib.rs

```diff
@@ -739,7 +757,7 @@ impl Runtime {
         // the Channel Label).
         let downgraded_source_label = self.get_node_downgraded_label(node_id, source_label);
         let target_label = self.get_node_label(node_id);
-        let node_name = self.get_node_name(node_id);
+        let node_name = self.get_node_debug_id(node_id);
```

Comment:
Nit! Could be fixed in the rest of the file too.
```suggestion
        let node_debug_id = self.get_node_debug_id(node_id);
```

Reply:
good point, done!